### DISCUSSION TITLE
The purpose of the /web/ route on subdomain

### DISCUSSION THREAD

**Discussion by retifrav** [umputun/remark42#1002] [Q&A] [answered]
I'm slightly confused about the `/web/` route on remark42 comments subdomain.  Let's say I allocated `comments.my-domain.com` for remark42, and so if I open `http://comments.my-domain.com/web/`, I get a kind of a demo page. Is it really for demo purposes only? So I can just disable this route in my web-server (*I run remark42 behind NGINX as reverse-proxy*)?

And second related question: there is something wrong with requests to server on that page. In web-inspector I can see requests to `http://127.0.0.1:8080/web/embed.mjs` and `http://127.0.0.1:8080/web/counter.mjs`, and of course they are failing.

The way I run remark42 in that case is:

``` sh
$ remark42.linux-amd64 server --secret=somesecret --url=https://comments.my-domain.com --port=8877 --site=some
```

so I am confused about the fact that it still tries to reach `127.0.0.1:8080`, even though that at the very least I provided a different port.

Comments on the main website work just fine, so this is only about about `/web/` route on subdomain.

**Comment by umputun** [comment, marked as answer]
> Is it really for demo purposes only?

yes. the practical usage is not to demo but rather a quick way to realize all configured properly

> so I am confused about the fact that it still tries to reach 127.0.0.1:8080, even though that at the very least I provided a different port.

likely a bug related to https://github.com/umputun/remark42/issues/995

**Reply by retifrav** [reply]
> a quick way to realize all configured properly

I've spent quite some time figuring out my configuration exactly because of things not working on that page, until I just checked my main website, where everything was working fine :)
But anyway, that's a minor issue really.

> likely a bug related to #995

Indeed, looks very similar. I'll track it there then. Thank you.

**Reply by akellbl4** [reply]
Problem with IP instead of the configured domain in your case connected to binary.
When we run Remark42 in docker we use `sed` to replace `127.0.0.1` on the configured domain. In case of binary we don't have that step.
We have a solution for this thing and it waits for implementation.
